frontend/www/js/omegaup/components/login/Signup.vue

```diff
@@ -117,6 +121,7 @@ import T from '../../lang';
 })
 export default class Signup extends Vue {
   @Prop() validateRecaptcha!: boolean;
+  @Prop() hasVisitedSection!: string;
```

Comment:
Ups, se nos había pasado esto. El tipo correcto es `boolean`:
```suggestion
  @Prop() hasVisitedSection!: boolean;
```